### ISSUE TITLE
Drawings on the map: Ground overlays

### ISSUE THREAD

**Issue by hemangsk** [hemangsk/capacitor-googlemaps-native#2]

**Comment by MatKershaw3708** [comment]
Is there any progress on adding overlays?

**Comment by BillyWizz68** [comment]
Yes, I am also interested in the map overlays. Is there a realistic timescale for it being ready? We would like to move over to Capacitor, but this is the one missing feature and is beyond our skills to create. Thank you for all your hard work and time.
